spec/classes/agent_spec.rb

```diff
@@ -409,6 +409,18 @@
           it { is_expected.to contain_file(config_path).with_content %r{^LogFileSize=100$} }
         end
       end
+
+      context 'when declaring manage_choco is false without zabbix_package_source specified' do
+        if facts[:kernel] == 'windows'
+          let :params do
+            {
+              manage_choco: false
+            }
+          end
+
+          it { is_expected.to compile.and_raise_error(%r{/expects a match for Stdlib::Windowspath/}) }
```

Comment:
Test that `package directive exists:

```suggestion
          it do
             is_expected.to contain_package(package_name).
               with_ensure('present').
               with_tag('zabbix').
               with_provider('windows').
               with_source(<source you set above>)               
          end 
```